src/Leaflet.Editable.js

```diff
@@ -1882,6 +1882,7 @@
         this.on('add', this._onEditableAdd);
     };
 
+    var flat = L.LineUtil._flat ? L.LineUtil._flat : L.Polyline._flat;  // 1.0/1.1 compat.
```

Comment:
Oh, yeah, forgot that `Leaflet` 1.1.0 already introduced the `L.LinueUtil._flat` 👍 
Great, that is great news, I hope the new Leaflet release won't make us wait too long 😄